modules/web/src/resource/config/configmap/detail/component.ts

```diff
@@ -66,4 +66,20 @@ export class ConfigMapDetailComponent implements OnInit, OnDestroy {
 
     return JSON.stringify(cm.data);
   }
+
+  getConfigMapBinaryData(cm: ConfigMapDetail): {
+    name: string;
+    size: number;
+  }[] {
+    if (!cm) {
+      return [];
+    }
+
+    return Object.entries(cm.binaryData).map(([name, value]) => ({
+      name,
+      size: atob(value).length,
```

Comment:
If we display only the length of it then I think it should be calculated on the server side.

Reply:
Thanks. Fixed in the latest commit